DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -148,7 +148,11 @@ void DmlDispatchable::Initialize()
         FillBindingData(m_desc.bindPoints.inputs, &m_initBindings, nullptr, inputBindingData, true);
 
         DML_BUFFER_ARRAY_BINDING bufferArrayBindings = {};
-        bufferArrayBindings.BindingCount = inputBindingData.bufferBindings.size();
+        if(inputBindingData.bufferBindings.size() > std::numeric_limits<uint32_t>::max())
+        {
+            throw std::invalid_argument(fmt::format("Init BindingCount '{}' is too large.", inputBindingData.bufferBindings.size()));
```

Comment:
"Init BindingCount" - did you mean "Input BindingCount" here? (or Initialize BindingCount?)

Reply:
It looks like this binding is for initialize I can change to Initialize Input    // Inputs flagged OWNED_BY_DML must be bound during initialization (and only initialization).
    if (!m_initBindings.empty())

Reply:
Mainly I just wondering whether "Init" (the tiny word fragment) was either a typo (and that you meant to type "Input") or an abbreviation for "Initialization" (in which case we should just spell it out in the error message).